pkg/controller/hostpathprovisioner/daemonset.go

```diff
@@ -584,7 +604,13 @@ func (r *ReconcileHostPathProvisioner) createCSIDaemonSetObject(cr *hostpathprov
 							TerminationMessagePolicy: corev1.TerminationMessageReadFile,
 						},
 						{
-							Name:            nodeDriverRegistrarName,
```

Comment:
The const is still there though? not sure why?

Reply:
rebase failure on my part